scripts/job_manager.ps1

```diff
@@ -0,0 +1,94 @@
+class MyJobManager {
+    [String] $Name = "JobManager"
+    [Hashtable] $Jobs = @{}
+
+    JobManager () {
+    }
+
+    [void] AddJob ($Tracker, $ScriptBlock, $ArgumentList, $InitScript) {
+        $job = Start-Job -ScriptBlock $ScriptBlock `
+                         -ArgumentList $ArgumentList `
+                         -InitializationScript $InitScript
+        Start-Sleep -Milliseconds 100
+        if ($job.State -ne "Running" ) {
+            Write-Host "Job failed to start" -ForegroundColor Red
+            $output = Receive-Job -Job $job
+            Remove-Job -Job $job
+            Write-host "Job failed with output `r`n$output`r`n" -ForegroundColor Red
+            return
+        }
+        if ($this.Jobs.Contains($Tracker)) {
+            $this.Jobs[$Tracker].AddLast($job)
+        } else {
+            $this.Jobs[$Tracker] = New-Object Collections.Generic.LinkedList[object]
+            $this.Jobs[$Tracker].AddLast($job)
+        }
+    }
+
+    [void] RemoveTracker ($Tracker) {
+        Write-Host "Removing jobs from tracker: $Tracker"
+        Start-Sleep 5
```

Comment:
ditto

Reply:
From a general perspective, start sleeps are an obvious lack of retries

Reply:
previously i kill the lisa process, and the job doesn't stop instantly
will try to stop the job first